README.md

```diff
@@ -371,6 +371,17 @@ const myBar = container.resolve<Bar>("Bar");
 You can also resolve all instances registered against a given token with `resolveAll()`.
 
 ```typescript
+interface Bar {}
+
+@injectable()
+class Foo implements Bar {}
+@injectable()
+class Baz implements Bar {}
+
+@registry([ // registry is optional, all you need is to use the same token when registering
```

Comment:
This usage of decorator is incomplete (it should be decorating a class). I think we need to fix this otherwise we could confuse some people

Reply:
done